dashboard/app/helpers/levels_helper.rb

```diff
@@ -594,4 +594,9 @@ def can_view_solution?
       Ability.new(current_user).can? :view_level_solutions, @script
     end
   end
+
+  def should_show_inline_answer?
+    current_user.try(:authorized_teacher?) && !@script.try(:pd?) ||
```

Comment:
Not certain of what you mean by this comment. Is it that I should be checking `professional_course?` instead of just `pd?`?